src/app/domains/User/components/combine/UserFilterProfile/UserFilterProfile.js

```diff
@@ -14,13 +14,13 @@ const tabList = [
   {
     tabName: 'Requests',
     icon: <PullRequestOutlined />,
-    content: <RequestList ownRequests />,
+    content: <RequestList ownRequests withButtonAccept={false} />,
```

Comment:
U can remove this prop, if u need `false`  value for this prop. Every prop equal to false value by default